nixos/modules/security/hidepid.nix

```diff
@@ -3,18 +3,23 @@ with lib;
 
 {
   options = {
-    security.hideProcessInformation = mkEnableOption "" // { description = ''
-      Restrict access to process information to the owning user.  Enabling
-      this option implies, among other things, that command-line arguments
-      remain private.  This option is recommended for most systems, unless
-      there's a legitimate reason for allowing unprivileged users to inspect
-      the process information of other users.
+    security.hideProcessInformation = mkOption {
+      default = true;
+      example = false;
+      description = ''
```

Comment:
Ah, this reminds me I should have moved this description to separate module documentation. I sometimes forget that the option description field isn't a blog :)